src/components/Toolbar/setAngleLength.tsx

```diff
@@ -24,51 +20,10 @@ import { normaliseAngle } from '../../lib/utils'
 import { kclManager } from 'lib/singletons'
 import { err } from 'lib/trap'
 import { KclCommandValue } from 'lib/commandTypes'
+import { angleLengthInfo } from './angleLengthInfo'
 
 const getModalInfo = createSetAngleLengthModal(SetAngleLengthModal)
 
-export function angleLengthInfo({
```

Comment:
was getting some circular dep issues when importing this into segments, so just moved it into it's own module.